doc/user/bgp.rst

```diff
@@ -2573,6 +2573,20 @@ are reached using *core* MPLS labels which are distributed using LDP or BGP
 labeled unicast.  *bgpd* also supports inter-VRF route leaking.
 
 
+L3VPN over GRE interfraces
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to configure and assign an incoming route-map to L3VPN peer,
+that permits BGP updates to be installed over a GRE interface instead of an
```

Comment:
Are you saying that a BGP adjacency is over the GRE or that the  next hop is over the GRE tunnel.  I suspect you mean the latter, but the text says the former.  Can you clarify?

Reply:
see previous command.